examples/TargetingConsoleApp/Program.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Consoto.Banking.AccountService.Identity;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.FeatureManagement;
+using Microsoft.FeatureManagement.FeatureFilters;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Consoto.Banking.HelpDesk
+{
+    class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            //
+            // Setup configuration
+            IConfiguration configuration = new ConfigurationBuilder()
+                .AddJsonFile("appsettings.json")
+                .Build();
+
+            //
+            // Setup application services + feature management
+            IServiceCollection services = new ServiceCollection();
+
+            services.AddSingleton(configuration)
+                    .AddFeatureManagement()
+                    .AddFeatureFilter<ContextualTargetingFilter>();
+
+            IUserRepository userRepository = new InMemoryUserRepository();
```

Comment:
Are some files missed from the PR?

Reply:
Never mind. Found them in a different folder :)

---
In reply to: [409118459](https://github.com/microsoft/FeatureManagement-Dotnet/pull/56#discussion_r409118459) [](ancestors = 409118459)